analyzer/src/main/kotlin/managers/utils/PythonInspector.kt

```diff
@@ -22,12 +22,28 @@ package org.ossreviewtoolkit.analyzer.managers.utils
 import com.vdurmont.semver4j.Requirement
 
 import java.io.File
+import java.io.FileInputStream
+import java.util.SortedSet
 
+import kotlinx.serialization.ExperimentalSerializationApi
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
-
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeFromStream
+
+import org.ossreviewtoolkit.analyzer.PackageManager
+import org.ossreviewtoolkit.model.Hash
+import org.ossreviewtoolkit.model.HashAlgorithm
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.Package
+import org.ossreviewtoolkit.model.PackageReference
+import org.ossreviewtoolkit.model.RemoteArtifact
+import org.ossreviewtoolkit.model.VcsInfo
+import org.ossreviewtoolkit.model.VcsType
 import org.ossreviewtoolkit.utils.common.CommandLineTool
-import org.ossreviewtoolkit.utils.common.ProcessCapture
+import org.ossreviewtoolkit.utils.ort.createOrtTempFile
+
+private val json = Json { ignoreUnknownKeys = true }
```

Comment:
This should be public, in a companion object, and be reused in the test (compare my earlier remark, and see existing similar code).

Reply:
It's moved here from the test class in this commit, no code outside of this class will deserialize python-inspector output directly.